tests/Cache/Subscription/PurgeSubscriptionProviderTest.php

```diff
@@ -305,7 +305,7 @@ class: 'FooEntity',
             });
 
         $managerRegistry = $this->createMock(ManagerRegistry::class);
-        $managerRegistry->method('getManager')
+        $managerRegistry->method('getManagerForClass')
```

Comment:
```suggestion
        $managerRegistry->expects(self::once())
            ->method('getManagerForClass')
```
maybe we should add `expects`? test would've failed if we had this before 😄

Reply:
Good idea, done